packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/Appointment.java

```diff
@@ -13,8 +13,6 @@ public class Appointment extends AbstractEntity {
     private String name;
     @NotBlank
     private String doctor;
-    @NotNull
```

Comment:
Because the field wasn't generated anymore now that the property is of type `datetime`. I figured since auto form doesn't support date / time fields, it would be OK to remove it for now.

Reply:
Ah ok, it can be added back later then